public/templates/index.html

```diff
@@ -124,21 +124,21 @@ <h5>For Practioners:</h5>
                     <p>If you are a practitioner sign up to offer free assistance to those in need. All credentials will be verified. </p>
                     <ul>
                         <li>
-                            <img src="/assets/images/about-icon-01.png" alt="">
+                            <img src="/public/assets/images/images/about-icon-01.png" alt="">
                             <div class="text">
                                 <h6>Get Verified</h6>
                                 <p>To sign up as a medical practioner, you must upload photocopies of your medical license, degree and other important credentials. Our team at Melly will then verify your credentials.</p>
                             </div>
                         </li>
                         <li>
-                            <img src="/assets/images/about-icon-02.png" alt="">
+                            <img src="/public/assets/images/images/about-icon-02.png" alt="">
```

Comment:
This is incorrect. The files were moved to "/public/assets/images" and there is no `images` sub-folder under that folder.

So it should be `/public/assets/images/...` and ditto elsewhere.